content/BindingMethods/BindingMethods.q.md

```diff
@@ -0,0 +1,1656 @@
+
+- `?`
+
+- `$sql`
+
+- `students`
+
+- `DELETE FROM`
+
+- `first_name = ?`
+
+
+/// type=SS, answer=[4]
+
+In the statement `$pstmt->bindValue(1, 'John');` on line 8 of `student.php`, what is `1`?
+
+- It represents the named placeholder.
+
+- It represents the explicit data type for the parameter.
+
+- It represents the value to bind to the positional placeholder `?` in the SQL statement. 
+
+- It represents the 1-index position of the positional placeholder `?` in the SQL statement.
+
+- It represents the variable to bind to the positional placeholder `?` in the SQL statement. 
+
+
+/// type=SS, answer=[4]
+
+In the statement `$pstmt->bindValue(1, 'John');` on line 8 of `student.php`, what does `John` represent?
+
+- The named placeholder.
+
+- The explicit data type.
+
+- The 1-index position of the parameter.
+
+- The value to bind to the positional placeholder `?`.
+
+- The variable to bind to the positional placeholder `?`.
+
+
+/// type=MS, answer=[3,4]
+
+Which statements correctly describe `1` and `John` on line 8 of `student.php`?
+
+- It contains the key and array value of the parameter.
+
+- It contains the key and array element of the parameter.
+
+- It represents the arguments of the `bindValue()` method.
+
+- It contains the 1-index position and value of the parameter.
+
+- It contains the variable and constant data type of the parameter.
+
+
+/// type=SS, answer=[2]
+
+In `student.php`, what does `bindValue()` do on line 8?
+
+- It binds the value `John` corresponding to the positional placeholder `?` in `last_name = ?` of the SQL statement.
+
+- It binds the value `John` corresponding to the positional placeholder `?` in `first_name = ?` of the SQL statement.
+
+- It assigns the value `John` corresponding to the positional placeholder `?` in `last_name = ?` of the SQL statement.
+
+- It replaces the value `John` corresponding to the positional placeholder `?` in `last_name = ?` of the SQL statement.
+
+- It replaces the value `John` corresponding to the positional placeholder `?` in `first_name = ?` of the SQL statement.
+
+
+/// type=MS, answer=[2,4,5]
+
+Which statements correctly descibe `$pstmt->bindValue(1, 'John');` on line 8 of `student.php`?
+
+- It executes the SQL statement.
+
+- It returns the boolean value `true`.
+
+- It assigns the value `John` to `$pstmt`.
+
+- It calls the `bindValue()` method of the `$pstmt` object.
+
+- It binds the value `John` corresponding to the positional placeholder `?` in `first_name = ?` of the SQL statement.
+
+
+/// type=SS, answer=[1]
+
+In `student.php`, which of the following represents the 1-index position as the parameter identifier on line 9?
+
+- `2`
+
+- `->`
+
+- `Smith`
+
+- `$pstmt`
+
+- `bindValue()`
+
+
+/// type=SS, answer=[1]
+
+In `student.php`, what does `bindValue()` do on line 9?
+
+- It binds the value `Smith` corresponding to the positional placeholder `?` in `last_name = ?` of the SQL statement.
```

Comment:
Please check this one also based on my comment on line 502. Please do the same to the rest of the options here... Thanks.